### PR TITLE
Fix signing files used by Profiler Attach

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
Add the following files to be signed:
* Microsoft.InstrumentationEngine.Attach.exe

### CODE REVIEW

File: src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj [hunk 1]
```diff
@@ -46,4 +47,8 @@
     <Error Condition="'%(XsdSchemaFile.Namespace)' == ''" Text="Missing Namespace attribute for XsdSchemaFile %(XsdSchemaFile.FullPath)" />
     <Exec Command="&quot;$(SDK40ToolsPath)\xsd.exe&quot; &quot;%(XsdSchemaFile.FullPath)&quot; /classes /namespace:%(XsdSchemaFile.Namespace) /language:%(XsdSchemaFile.Language) /out:$(MSBuildThisFileDirectory)" />
   </Target>
+  <ItemGroup>
+    <!-- dotnet core 3.1 produces both a dll and exe; project references only pick up the dll so this explicitly adds the exe for signing -->
+    <FilesToSign Include="$(OutputPath)\Microsoft.InstrumentationEngine.Attach.exe"/>
```
Comment (wiktork) [inline src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj:52]: This comment is confusing. I would expect the Microbuild targets file to sign the exe output as well. Whether other projects have references to this one seems orthogonal.

Comment (WilliamXieMSFT) [reply src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj:52]: Ah, the comment was copied from Aerobee which uses a signproj with a projectreference. I had assumed this was what caused the dll to get signed and not the exe, hence the comment.

File: build/CodeSign/certificate.ignore [hunk 2]
```diff
@@ -15,4 +15,6 @@ bin\*\*.wixlib, Ignore wix artifact
 bin\*\*.wixpdb, Ignore wix artifact
 bin\*\*.msidir\*.cab, Ignore embedded cab files
 
-binaries-windows-release\*, Ignore downloaded binary artifacts
\ No newline at end of file
+binaries-windows-release\*, Ignore downloaded binary artifacts
```
Comment (jander-msft) [inline build/CodeSign/certificate.ignore:18]: I think you can revert all of the exclusions if my XDT extensions signing PR is completed.
